src/Vote.Monitor.Api.Feature.Observer/Services/CsvParser.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System.Globalization;
+using CsvHelper;
+using CsvHelper.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Vote.Monitor.Api.Feature.Observer;
+using Vote.Monitor.Api.Feature.Observer.Services;
+
+namespace Vote.Monitor.Api.Feature.PollingStation.Services;
+
+
+
+public class CsvParser<T, TMapper> : ICsvParser<T> where T : class, IDuplicateCheck where TMapper : ClassMap<T>
```

Comment:
This is the first use case when we need to make sure rows are unique I will suggest to remove generics.